bin/import-file

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+# Contains logic to import files regardless of how many there are.
+# If there's no file, don't do anything with the database.
+# If the table already exists in the database, don't try to re-create it.
+#
+# Usage:
+#   bin/import-file [csv_path] [table]
+#   bin/import-file downloads/csv A-Contributions
+set -euo pipefail
+
+if [ -z "${DATABASE_NAME:-""}" ]; then
+  echo "Error: Set DATABASE_NAME."
+  exit 1
+fi
+
+if [ $# -eq 0 ]; then
+  echo "Usage: bin/import-file [csv_path] [table]"
+  exit 1
+else
```

Comment:
This can just be an `fi` since it'll exit if the guard catches.

Reply:
Good catch. Fixed in an upcoming commit.